Sources/Core/Selection/FilterSelectionStore.swift

```diff
@@ -81,6 +86,27 @@ extension FilterSelectionStore {
     }
 }
 
+private extension FilterSelectionStore {
+    func _setValue<T: LosslessStringConvertible>(_ value: T?, for filter: Filter) {
+        removeValues(for: filter)
```

Comment:
Shouldn't you call `_removeValues` here?

Reply:
Yes! It should :P